src/Common/src/System/Globalization/IdnMapping.Unix.cs

```diff
@@ -11,44 +11,37 @@ private unsafe string GetAsciiCore(char* unicode, int count)
             uint flags = Flags;
             CheckInvalidIdnCharacters(unicode, count, flags, nameof(unicode));
 
+            int length = Interop.GlobalizationNative.ToAscii(flags, unicode, count, null, 0);
```

Comment:
Thanks - I've done as you suggested. 

I tried this with Windows as well, but alas the Windows APIs return 0 not the actual length if the buffer overflows

Is there a way to disable this? I don't think so